optimum/exporters/openvino/model_patcher.py

```diff
@@ -2991,3 +2991,25 @@ def __init__(
     def __exit__(self, exc_type, exc_value, traceback):
         super().__exit__(exc_type, exc_value, traceback)
         self._model.forward = self._model.__orig_forward
+
+
+class InputEmbeddingPatcher(ModelPatcher):
```

Comment:
This seems like a quite general patcher. Its name probably should not reflect that it is currently applied to input embeddings, but that it patches forward method to have a single input.

Reply:
I do not think that it can be generalized to other modules, because it requires to know original signature in  other modules forward it can be different, so current patcher implementation will not fit to these purpoceses. I do not want to make it generic too, because there is opportunity to fix it on openvino frontend level